pkg/tagset/composite_tags.go

```diff
@@ -61,6 +62,16 @@ func (t *CompositeTags) CombineWithSlice(tags []string) {
 	*t = CombineCompositeTagsAndSlice(*t, tags)
 }
 
+// Tags1 gives access to the first set of tags
+func (t CompositeTags) Tags1() []string {
```

Comment:
Can't you just use `UnsafeToReadOnlySliceString` instead? (your FIXME in `encoding.go:44`)